### ISSUE TITLE
 Will the comments column be added?

### ISSUE THREAD

**Issue by theclanbaymis** [CalvinTan68/UOY---Anonymous-Chat#2]
Do you have in mind to add comments to the repo?

**Comment by CalvinTan68** [comment]
Yes i do have, even though i'm still not sure because it's a secret platform. You can clone this repository and try to add comments section 👍
